src/main.ts

```diff
@@ -19,19 +20,68 @@ import { WithdrawalFinalizeFlow } from "./withdrawalFinalize";
 const main = async () => {
   setupLogger(process.env.NODE_ENV, process.env.LOG_LEVEL);
   const l2Provider = new LoggingZkSyncProvider(unwrap(process.env.CHAIN_RPC_URL));
+  const l2EthersProvider = new JsonRpcProvider(unwrap(process.env.CHAIN_RPC_URL));
+  l2EthersProvider.pollingInterval = 100;
```

Comment:
If this provider was created just to override polling interval, why not override polling interval on the normal provider?